css/style.css

```diff
@@ -0,0 +1,2927 @@
+}
+
+.middle__info-title {
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  margin-top: 51px;
+  margin-bottom: 12px;
+}
+
+.middle__info-text {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  margin-top: 11px;
+  margin-right: 70px;
+  margin-bottom: 50px;
+}
+
+.middle__row {
+  display: flex;
+  flex-direction: column;
+  padding-top: 5px;
+  padding-left: 45px;
+  padding-right: 150px;
+}
+
+.middle__btn {
+  margin-top: 90px;
+  margin-left: 8px;
+}
+
+.middle__column {
+  display: flex;
+  align-items: center;
+  margin-bottom: 6px;
+}
+
+.middle__column:nth-child(3) {
+  margin-top: 5px;
+}
+
+.middle__screen {
+  padding-top: 18px;
+  padding-left: 7px;
+  padding-bottom: 3px;
+}
+
+.middle__wifi {
+  padding-top: 0;
+  padding-bottom: 0;
+  margin-left: 3px;
+  margin-top: 1px;
+}
+
+.middle__bag {
+  padding-right: 4px;
+  padding-top: 18px;
+  padding-left: 7px;
+  padding-bottom: 0;
+}
+
+.middle__item {
+  height: 90px;
+}
+
+.middle__first {
+  padding-top: 5px;
+  padding-left: 0px;
+}
+
+.middle__first-big {
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+  padding-left: 20px;
+  padding-bottom: 0;
+  padding-right: 7px;
+}
+
+.middle__first-small {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+  padding-left: 0;
+}
+
+.middle__first-title{
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 18px;
+  padding-left: 20px;
+  padding-bottom: 0;
+  padding-top: 8px;
+}
+
+.middle__first-text {
+  font-size: 12px;
+  line-height: 15px;
+  padding-left: 20px;
+  padding-bottom: 10px;
+}
+
+.middle__text {
+  font-size: 12px;
+  line-height: 15px;
+  padding-left: 20px;
+}
+
+.middle__btn {
+  margin-top: 95px;
+  margin-left: 12px;
+}
+
+.button__btn {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  border: 2px solid #E51937;;
+  color: #FFFFFF;
+  padding-left: 35px;
+  padding-right: 35px;
+  margin-bottom: 0;
+  margin-right: 10px;
+  background-color: #E51937;
+  margin-left: 13px;
+  margin-top: 148px;
+}
+
+.button__btn-img {
+  padding-right: 7px;
+}
+
+.button__btn-word {
+  font-family: Montserrat;
+  font-style: normal;
+  text-align: center;
+  font-weight: 700;
+  font-size: 13px;
+  padding-right: 0;
+  margin-right: 0;
+  width: 125px;
+}
+
+.button__btn-text {
+  padding: 10px 28px;
+  border: 2px solid #ffffff;
+  font-size: 13px;
+  text-transform: uppercase;
+  color: #ffffff;
+  background-color: #E51937;
+  font-family: Montserrat;
+  font-style: normal;
+  text-align: center;
+  font-weight: 700;
+  font-size: 13px;
+  padding-left: 40px;
+  padding-right: 40px;
+  margin-left: 33px;
+  margin-right: 3px;
+}
+
+.bottom {
+  background: url(../img/model-s-bottom.png) no-repeat;
+  background-position: 647px 142px;
+}
+
+.bottom__inner {
+  display: flex;
+}
+
+.bottom__info {
+  display: flex;
+  flex-direction: column;
+  width: 440px;
+  height: 900px;
+  color: #ffffff;
+  background: #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)